From fb3ba9af37feadb2b024479421bef2d7199c9271 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Thu, 12 Mar 2015 15:58:50 +0800
Subject: [PATCH] MLK-10411 ARM: dts: imx7d-12x12-arm2: enable pxp v4l2 output

Enable the pxp v4l2 output module for imx7d 12x12 board
in the related dts file.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-arm2.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-arm2.dts
index bb791db..2076bef 100644
--- a/arch/arm/boot/dts/imx7d-12x12-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-arm2.dts
@@ -41,6 +41,11 @@
 		};
 	};
 
+	pxp_v4l2_out {
+		compatible = "fsl,imx7d-pxp-v4l2", "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
 	regulators {
 		compatible = "simple-bus";
 		#address-cells = <1>;
-- 
1.8.0

